src/test-data.csv

```diff
@@ -0,0 +1,87 @@
+Day,Time,Track 1.language,Track 1.title,Track 1.description,Track 1.type,Track 1.duration (min) ,Track 2.language,Track 2.title,Track 2.description,Track 2.type,Track 2.duraion (min) ,Track 3.language,Track 3.title,Track 3.description,Track 3.type,Track 3.duration (min
```

Comment:
Don't include this file and it should be private. Instead add mock version.

Reply:
we'll just set the source in the .env file